### DISCUSSION TITLE
Is it possible to setup more than one firmware repositories ?

### DISCUSSION THREAD

**Discussion by vijay-bhakta** [fwupd/fwupd#5751] [General]
I came across article #4209 which might be helpful if we try and plan to setup a private repo.

But, is it possible that 
`fwupdmgr get-updates`
`fwupdmgr update`
refer to our own repo along with LVFS standard repo for new firmwares?

**Comment by hughsie** [comment]
> if we try and plan to setup a private repo

Why would you need that? Does an embargo remote on the main LVFS not work for testing? See https://lvfs.readthedocs.io/en/latest/testing.html#end-to-end-testing for the documentation -- lots of vendors use the embargo remotes to test in private without the firmware ever being released to the public.

**Reply by vijay-bhakta** [reply]
Some of vendors mentioned supporting DFU , but still has doubts to ship via LVFS.  Don't know the reasons, and may get insights from more discussions with them. May be , firmware not fuly conforming to DFU spec , who knows.
But, if they are and its only about confidence, I want to enable shippping with other repo for such vendors for short term.

My understanding from the link provide by you is that we can receive embargo configuration file from vendor and with that we can enable LVFS in host device to access firmware under beta testing.
correct?

**Reply by hughsie** [reply]
>  Don't know the reasons, and may get insights from more discussions with them

Okay, I'm happy to help cut through the real reasons if you want to add me to the email cc.

> My understanding from the link provide by you is that we can receive embargo configuration file from vendor

Yes, or the vendor can share the firmware directly with your vendor too.

**Reply by vijay-bhakta** [reply]
@hughsie 
Just side question , do you provide some kind of awareness session on this technology as guest speaker to any company interested ?  :)

**Reply by hughsie** [reply]
Ha, I'm not sure it qualifies as a _session_, but I'd gladly talk to any vendor that wants to listen about LVFS and fwupd. I've got various slide decks I use for high level concepts for management and low-level deep dives for firmware teams. My email is richard_at_hughsie.com and I normally work GMTish 9am-5pm as I'm in London.

**Reply by vijay-bhakta** [reply]
Well , we are not vendors but consumers of LVFS. But, high level concepts for management may help how we promote LVFS technology to our partners(USB device vendors)

**Reply by hughsie** [reply]
Fine for me! My hughsient_at_gmail.com calendar is public if you want to put something in. Please tell me how long you'd like me to speak for, and what the intended audience is.

**Reply by vijay-bhakta** [reply]
Thanks. Will let you know when need arise.
